ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestExecutorUtil.java

```diff
@@ -268,7 +273,8 @@ private static void initializeTopics(
       final StubKafkaService stubKafkaService,
       final TestExecutionListener listener
   ) {
-    final ImmutableList.Builder<PersistentQueryAndSources> queriesBuilder = new Builder<>();
+    final Map<QueryId, PersistentQueryAndSources> queries = new HashMap<>();
```

Comment:
This will likely results in the _order_ of entries in the returned list changing from what it was before, and not being non-deterministic. 

Not sure if this is important, just flagging as a potential unforeseen side effect of the change.

Reply:
good call, I will change this as I think it might be what's actually causing the QTT for some other things to fail